developer_manual/app/schema.rst

```diff
@@ -1,3 +1,102 @@
+===================
+Database Migrations
+===================
+
+.. sectionauthor:: Thomas Müller <thomas.mueller@tmit.eu>
+
+ownCloud uses `Doctrine Migrations <http://www.doctrine-project.org/projects/migrations.html>`_ to manage any kind of database changes.
+
+Any app which wants to use this mechanism has to enable it in :file:`appinfo/info.xml` by adding
+
+.. code-block:: xml
+
+  <use-migrations>true</use-migrations>
+
+
+As soon as migrations are enabled any existing :file:`appinfo/database.xml` will be ignored on upgrade of the app.
+Instead, all migration steps which have not yet been executed will be executed.
```

Comment:
@DeepDiver1975 this doesn't fully make sense. The previous sentence says that they'll be ignored; yet this one says that any outstanding ones will be executed. Can you clarify pls?